### DISCUSSION TITLE
Instructions on Wiki unclear for moving scenes.

### DISCUSSION THREAD

**Discussion by TheBitcoinBaron** [kinsi55/CS_BeatSaber_Camera2#49] [Q&A] [answered]
Copied the movement JSON to the folder from the wiki... but then what? How to make it work in game? There is no area from which to select the movement script in game.

**Comment by kinsi55** [comment]
Assigning Movement scripts cannot be done from Ingame but must be done manually in the respective Camera config, As outlined in the Camera Config wiki: https://github.com/kinsi55/CS_BeatSaber_Camera2/wiki/Camera-Configuration#movementscript-positionable-only

**Comment by TheBitcoinBaron** [comment]
I have saved the template movement file as a JSON in the directory indicated, but then I still have no idea what to do next to make this work. I'm sure it is maybe obvious, but I am pretty dumb. Can you spell it out or step-by step the rest of the process?

**Comment by kinsi55** [comment, marked as answer]
the [Example usage for Movement Scripts](https://github.com/kinsi55/CS_BeatSaber_Camera2/wiki/Movement-Scripts#example-movement-script-setup--configuration) link in the previous comments link leads to a pictured example config. I've also added text to that section now. Does that help?

**Comment by TheBitcoinBaron** [comment]
Ahh i get it, it was the adding of the "MovementScript" part that I somehow didn't even see. Makes sense now.
